modules/apps/frontend-js/frontend-js-module-launcher/package.json

```diff
@@ -0,0 +1,10 @@
+{
+	"main": "webpack_federation.js",
```

Comment:
We may want to remove this in the future, since it won't be used as we abandoned the webpack federation path.

Reply:
Thats fine by me, this was just an aim to make sure all our client served js is properly minified